Source/WebCore/dom/ContainerNode.h

```diff
@@ -78,12 +78,14 @@ class ContainerNode : public Node {
     struct ChildChange {
         enum class Type : uint8_t { ElementInserted, ElementRemoved, TextInserted, TextRemoved, TextChanged, AllChildrenRemoved, NonContentsChildRemoved, NonContentsChildInserted, AllChildrenReplaced };
         enum class Source : bool { Parser, API };
+        enum class AffectsElements : bool { No, Yes };
 
         ChildChange::Type type;
         Element* siblingChanged;
         Element* previousSiblingElement;
         Element* nextSiblingElement;
         ChildChange::Source source;
+        AffectsElements wereElementAffected;
```

Comment:
I'd call this member variable `affectedElements` or something instead.

Reply:
Ok.